lib/write-file.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+
+function uniqueFile(file) {
+  var testPath = file;
+  var counter = null;
+  while (fs.existsSync(testPath + '.png')) {
+    counter = (counter || 0) + 1;
+    testPath = file + counter;
+  }
+
+  return testPath + '.png';
+}
+
+function getFile(directory, title) {
+  // Take the test title and remove all special characters; limit to 20 characters
+  var name = title.replace(/[^\w]/g, '_').replace(/_{2,}/g, '_').substr(0, 40);
```

Comment:
I'm assuming that the code is right™, so I just made the comment less specific. Generally cleaned up that expression and moved the magic number to a named identifier.